yjit/src/stats.rs

```diff
@@ -354,6 +354,18 @@ fn rb_yjit_gen_stats_dict() -> VALUE {
         let key = rust_str_to_sym("outlined_code_size");
         let value = VALUE::fixnum_from_usize(ocb.unwrap().get_write_pos());
         rb_hash_aset(hash, key, value);
+
+        // GCed code size
+        #[cfg(feature = "stats")]
+        {
+            let key = rust_str_to_sym("inline_code_gc_size");
+            let value = VALUE::fixnum_from_usize(cb.freed_bytes);
+            rb_hash_aset(hash, key, value);
+
+            let key = rust_str_to_sym("outlined_code_gc_size");
+            let value = VALUE::fixnum_from_usize(ocb.unwrap().freed_bytes);
+            rb_hash_aset(hash, key, value);
+        }
```

Comment:
This stat is not expensive to collect so we could dispense with the complexity of adding `#[cfg(feature = "stats")]` in many places IMO, but I will leave that up to you.